.github/workflows/ci-core-reusable.yml

```diff
@@ -52,22 +52,17 @@ jobs:
 
       - name: Init
         run: |
-          ci_run zk
           ci_run run_retried rustup show
-          ci_run zk run yarn
-          ci_run zk db setup
-          ci_run zk compiler all
-          ci_run zk contract build
-
-      - name: Contracts unit tests
-        run: ci_run yarn l1-contracts test
+          ci_run yarn l1-contracts build
+          ci_run yarn l2-contracts build
```

Comment:
You have to install it with zk supervisor, and you also have to install  yarn. 
Moreover for l1 contracts we already use foundry, so please use it instead of hardhat